php_phongo.c

```diff
@@ -2375,8 +2375,19 @@ PHP_MINFO_FUNCTION(mongodb)
 	php_info_print_table_header(2, "mongodb support", "enabled");
 	php_info_print_table_row(2, "mongodb version", MONGODB_VERSION_S);
 	php_info_print_table_row(2, "mongodb stability", MONGODB_STABILITY_S);
+#ifdef HAVE_SYSTEM_LIBMONGOC
+	php_info_print_table_row(2, "libmongoc headers version", MONGOC_VERSION_S);
+	php_info_print_table_row(2, "libmongoc library version", mongoc_get_version());
+#else
```

Comment:
Ah ok. Makes sense. Shall we make that the third line? mongoc: system/bundled ?
Then its easy `ctrl+f`able, along with being straight forward to ask/answer the question